sb-form-server/src/jsdom/submissionView.js

```diff
@@ -11,13 +11,14 @@ const { JSDOM } = jsdom;
 const baseUrl = 'http://local';
 
 const render = async (templateName, submission) => new Promise((resolve, reject) => {
-  const rejectTimeout = setTimeout(() => (reject(new Error('Failed to render submission in jsdom'))), 5000);
+  // ! time needs to be addressed
+  const rejectTimeout = setTimeout(() => (reject(new Error('Failed to render submission in jsdom'))), 20000);
```

Comment:
Use 5000, also use the bad request trick above if poss to give the user a better experience? (if this is running after the response is returned, see if you can move it in. Ideally we want to feed back to the user.